### ISSUE TITLE
Mardown highlighting in comments consumes closing `-/`

### ISSUE THREAD

**Issue by eric-wieser** [leanprover/vscode-lean#274]
This isn't highlighted correctly:
```lean
/- Oh `-/` no -/
#check 1
```
The `no -/` is not actually part of the comment.

**Comment by gebner** [comment]
Yeah, I am well aware that this can happen.  I've been fixing them whenever they become an issue.  Unfortunately it seems to be impossible to tell vscode to stop the markdown highlighting when it sees a `-/`, the markdown grammar needs to cooperate here and stop itself...

In general, I didn't prioritize invalid markdown syntax, because you should usually avoid it.   See also https://github.com/leanprover-community/vscode-lean4/issues/10#issuecomment-796860331

**Comment by eric-wieser** [comment]
Perhaps I should have made my intention clear in opening this - I'm not after a fix, I figured it was just worth recording the bug in case someone else cares enough to fix it.

**Comment by eric-wieser** [comment]
IIRC, sublime text ended up augmenting their TextMate grammar handling to make this type of "just kill the inner parser if you find a `-/`" action possible. Perhaps vscode will eventually go the same way.
